docs/examples/feed-example.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "_id" : "video1",
-    "type" : "video",
+    "type" : "Video",
```

Comment:
Capitalized because RAML was complaining at me that my enum def had caps, so...no. I'll make them lower again.